src/EditorFeatures/Test/Diagnostics/DiagnosticDataSerializerTests.cs

```diff
@@ -271,7 +275,7 @@ private static void AssertDiagnostics(DiagnosticData item1, DiagnosticData item2
             Assert.Equal(item1.HelpLink, item2.HelpLink);
         }
 
-        [ExportWorkspaceServiceFactory(typeof(IPersistentStorageService), "DiagnosticDataSerializerTest"), Shared]
+        [ExportWorkspaceServiceFactory(typeof(IPersistentStorageService), WorkspaceKind.Host), Shared]
```

Comment:
❕ Exports in test code need to use `PartNotDiscoverable`

```suggestion
        [ExportWorkspaceServiceFactory(typeof(IPersistentStorageService), WorkspaceKind.Host), Shared]
        [PartNotDiscoverable]
```